driver/sdk.md

```diff
@@ -1,4 +1,135 @@
 
 # Babelfish SDK
 
-* **TODO**
\ No newline at end of file
+The [Babelfish SDK](https://github.com/bblfsh/sdk/) contains tools and libraries
+required to create a Babelfish driver.
+
+## Prerequisites
+
+Babelfish has the following prerequisites:
+
+* [Docker](https://www.docker.com/get-docker)
+* [Go](https://golang.org/dl/)
+
+## Installing
+
+Install the Babelfish SDK:
+
+```bash
+$ go get github.com/bblfsh/sdk/...
+```
+
+This will fetch the sdk repository to `$GOPATH/src/github.com/bblfsh/sdk` and
+will install the `bblfsh-sdk` CLI to `$GOPATH/bin/bblfsh-sdk`.
+
+## Bootstrapping a new driver
```

Comment:
May be "Create a driver for a new language" would be a bit less ambiguous as bootstrap? 

Otherwise it may have a co-notation on "bootstrapping a compiler" and may refer to self-hosting, which seems not to be the case here.